File: Dataset/Leetcode/test/58/232.java
 class Solution {
    public int XXX(String s) {
        int count = 0;
        for (int i = s.length() - 1; i >= 0; i--) {
            char c = s.charAt(i);
            if (c == ' ') {
                if (count > 0) {
                    return count;
                }
            } else {
                count++;
            }
        }
        return count;
    }
}

File: Dataset/Leetcode/train/66/232.cpp
 class Solution {
public:
    vector<int> XXX(vector<int>& digits) {
        vector<int> res{};
        int jinwei = 0;
        digits.back()++;
        while(!digits.empty())
        {
            int tmp = digits.back();
            digits.pop_back();
            tmp += jinwei;
            jinwei = tmp / 10;
            tmp %= 10;
            res.push_back(tmp);
        }
        if (jinwei)
        {
            res.push_back(1);
        }
        reverse(res.begin(),res.end());
        return res;
    }
};

